docs/en/operations/system-tables/information_schema.md

```diff
@@ -218,3 +226,62 @@ is_trigger_updatable:       NO
 is_trigger_deletable:       NO
 is_trigger_insertable_into: NO
 ```
+
+## KEY_COLUMN_USAGE (#key_column_usage)
+
+It was added for compatibility with third party tools such as Tableau Online. Contains only the primary keys columns read from [system.columns](../../operations/system-tables/columns.md).
+
+Columns:
+
+- `constraint_catalog` ([String](../../sql-reference/data-types/string.md)) — The name of the catalog to which the constraint belongs. This value is always `def`.
```

Comment:
There is no such thing as a "catalog" in ClickHouse. Let's not confuse the reader. You can just say

```md
- `constraint_catalog` ([String](../../sql-reference/data-types/string.md)) — Currently unused. Always `def`.
```